content/en/docs/ambient/usage/ztunnel/index.md

```diff
@@ -226,7 +226,7 @@ $ echo "$ZTUNNEL"
 {{< /text >}}
 
 {{< text bash >}}
-$ istioctl proxy-config workloads "$ZTUNNEL".istio-system
+$ istioctl x ztunnel-config workloads "$ZTUNNEL".istio-system
```

Comment:
Does user needs to specify "$ZTUNNEL".istio-system?  I don't think it is required.

Reply:
I don't think it's required, I can verify though. I was just going off of previous functionality which still seems to work.

Reply:
Let us keep the doc as simple as possible - It is not required so let us remove it?

Reply:
agreed!

Reply:
We also expand out x to experimental

Reply:
(I do wonder if the UX of having `istioctl x` is somewhat at odds with how we use feature statuses, and should be removed.  I'll bring this up on Slack)